wsd/DocumentBroker.cpp

```diff
@@ -1145,7 +1145,7 @@ void DocumentBroker::lockIfEditing(const std::shared_ptr<ClientSession>& session
         {
             LOG_DBG("Locking docKey [" << _docKey << "], which is editable");
             std::string error;
-            if (!updateStorageLockState(Authorization::create(uriPublic), error))
+            if (!lockDocumentInStorage(Authorization::create(uriPublic), error))
             {
```

Comment:
here you seem to change semantics? not update the lock state but lock it (initially?)
Intended? @Ashod (potential issue or a bug fix of yours?)

Ah, I see - you changed the name (below): `updateStorageLockState` -> `lockDocumentInStorage`. 
OK, assuming it matches semantics better.

Reply:
Good that you're checking/catching subtle things like this. I like your approach.

`updateStorageLockState` is (was) overloaded. The one with 2 parameters only locked, and was called only at the start (without a client session). This is just renaming that overload, since it was really locking and that is a more accurate name (also, overloading in this case seems unnecessary source of confusion).